app/views/topics/show.html.erb

```diff
@@ -1,6 +1,10 @@
 <% content_for :title do %><%= @topic.title %> - GOV.UK<% end %>
 <% content_for :page_class, "topics-page" %>
 
+<div class="header-context group">
```

Comment:
You don't actually need the 'group' class here. The 'group' class was a way we used to use to make sure elements 'wrap' any floated elements contained within them.